apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -2659,7 +2659,7 @@ export const extensions: Extensions = {
                 },
                 searchBar: {
                     placeholderDiagnostic: "Search Logs by Trace ID, Action ID, Client ID, Result Message, or Result Status",
-                    placeholderAudit: "Search Logs by Action, Target ID, Initiator ID, Request ID"
+                    placeholderAudit: "Search Logs by actionId, targetId, initiatorId, traceId"
```

Comment:
Is this change intentional? IMO, we should keep the existing text content for better UX.

Reply:
Yes. Intentional. We are aligning with the body of the logs. **Action** have no meaning when the logs only have **actionId**. And can you please explain what it means by `we should keep the existing text content for better UX.`